src/drizzle/0000_typeorm_state.sql

```diff
@@ -0,0 +1,447 @@
+ALTER TABLE "item_tag" ADD CONSTRAINT "FK_16ab8afb42f763f7cbaa4bff66a" FOREIGN KEY ("tag_id") REFERENCES "public"."tag"("id") ON DELETE cascade ON UPDATE no action;--> statement-breakpoint
+ALTER TABLE "item_tag" ADD CONSTRAINT "FK_39b492fda03c7ac846afe164b58" FOREIGN KEY ("item_id") REFERENCES "public"."item"("id") ON DELETE cascade ON UPDATE no action;--> statement-breakpoint
+CREATE INDEX "IDX_6079b3bb63c13f815f7dd8d8a2" ON "app_data" USING btree ("type" text_ops);--> statement-breakpoint
+CREATE INDEX "IDX_account_type" ON "account" USING btree ("type" text_ops);--> statement-breakpoint
+CREATE INDEX "IDX_1214f6f4d832c402751617361c" ON "action" USING btree ("item_id" uuid_ops);--> statement-breakpoint
+CREATE INDEX "IDX_action_account_id" ON "action" USING btree ("account_id" uuid_ops);--> statement-breakpoint
+CREATE INDEX "IDX_bdc46717fadc2f04f3093e51fd" ON "item" USING btree ("creator_id" uuid_ops);--> statement-breakpoint
+CREATE INDEX "IDX_gin_item_search_document" ON "item" USING gin ("search_document" tsvector_ops);--> statement-breakpoint
+CREATE INDEX "IDX_gist_item_path" ON "item" USING gist ("path" gist_ltree_ops);--> statement-breakpoint
+CREATE INDEX "IDX_61546c650608c1e68789c64915" ON "app_setting" USING btree ("item_id" uuid_ops,"name" text_ops);--> statement-breakpoint
+CREATE INDEX "IDX_gist_item_published_path" ON "item_published" USING gist ("item_path" gist_ltree_ops);--> statement-breakpoint
+CREATE INDEX "IDX_5ac5bdde333fca6bbeaf177ef9" ON "item_membership" USING btree ("permission" text_ops);--> statement-breakpoint
+CREATE INDEX "IDX_d935785e7ecc015ed3ca048ff0" ON "item_membership" USING btree ("item_path" ltree_ops);--> statement-breakpoint
+CREATE INDEX "IDX_gist_item_membership_path" ON "item_membership" USING gist ("item_path" gist_ltree_ops);--> statement-breakpoint
+CREATE INDEX "IDX_item_membership_account_id" ON "item_membership" USING btree ("account_id" uuid_ops);--> statement-breakpoint
+CREATE INDEX "IDX_item_membership_account_id_permission" ON "item_membership" USING btree ("account_id" uuid_ops,"permission" text_ops);--> statement-breakpoint
+CREATE INDEX "IDX_gist_item_visibility_path" ON "item_visibility" USING gist ("item_path" gist_ltree_ops);--> statement-breakpoint
+CREATE INDEX "IDX_43c8a0471d5e58f99fc9c36b99" ON "short_link" USING btree ("item_id" uuid_ops);--> statement-breakpoint
+CREATE INDEX "IDX_91fa43bc5482dc6b00892baf01" ON "member_profile" USING btree ("member_id" uuid_ops);--> statement-breakpoint
+CREATE INDEX "IDX_item_like_item" ON "item_like" USING btree ("item_id" uuid_ops);--> statement-breakpoint
+CREATE INDEX "IDX_item_tag_item" ON "item_tag" USING btree ("item_id" uuid_ops);
+
+-- -- -- -- -- -- -- -- -- -- -- -- -- --
+-- TYPEORM MIGRATION - Comment up to here
+-- -- -- -- -- -- -- -- -- -- -- -- -- --
+
+-- Keep this is you migrate from typeorm
+create table "test" (id text not null);--> statement-breakpoint
+drop table "test";
```

Comment:
creating a table and dropping it right away? 
Should we have this in another sql migration?

Reply:
Yes this is just some dummy code for when the whole migration is commented, otherwise drizzle complains and does not register the migration file.